README.md

```diff
@@ -1,134 +1 @@
-  - [EazeGraph](https://github.com/blackfizz/EazeGraph) - EazeGraph is an Android library for creating beautiful and fancy charts. 
-  - [fab-speed-dial](https://github.com/yavski/fab-speed-dial) - A simple library marrying together [FAB]
-
-## Screenshots 📷 
-![light_screenshot](https://github.com/sitamadex11/CovidHelp/blob/develop/Assets/AllScreenShots.png)
-
-## Download 📥
-- Click [Here](https://drive.google.com/u/2/uc?id=1OWIi9xcCLTTyZtJHTVzPAjbA3RiiVCal&export=download) to download the latest apk.
-
-## Requirements 🎯 
-- Android 6.0 and Above
-- Min sdk version 23
-
-## Permissions 💻
-- Internet
-
-## Contributing Guideline ✍
-- Before start your contribution please go through our contribution guideline carefully. ([READ](https://github.com/sitamadex11/CovidHelp/blob/develop/Assets/Contributing.md))
-- You need to push your PR on [develop](https://github.com/sitamadex11/CovidHelp/tree/develop) branch only.
-
-## 😎Maintainers
-<table>
-  <tbody><tr>
-    <td align="center"><a href="https://github.com/sitamadex11"><img alt="" src="https://avatars.githubusercontent.com/sitamadex11" width="200px;"><br><sub><b> Sitam Sardar </b></sub></a><br></td> </a></td>
-
-     
-    
-  </tr>
-</tbody></table>
-
-## License 
-
-```
-MIT License
-
-Copyright (c) 2021 Sitam Sardar
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-```
-### If you liked the project don't forget to star 🌟 and fork 🍽 the project.
-![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)
-![ForTheBadge ANDROID](https://forthebadge.com/images/badges/built-for-android.svg)
-![ForTheBadge GIT](https://forthebadge.com/images/badges/uses-git.svg)
-
-### Credits goes to these people:✨
-
-<table>
-	<tr>
-		<td>
-   <a href="https://github.com/sitamadex11/CovidHelp/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=sitamadex11/CovidHelp" />
-</a>
-		</td>
-	</tr>
-</table>
-
-
-
-<br>
-<br>
-<h2 align="center">❤ Our Aim to Help People ❤</h2>
-
-<br>
+# CovidHelp
```

Comment:
Hey, why are you deleting the whole readme file?

Reply:
Oh I am not deleting anything I took the commit from the upstream branch

Reply:
sorry I merged all commits into develop wait I ll revert and proper merge to develop only

Reply:
and I will update gitignore file too coz it is taking idea and other cache files from my repo which it shouldn't will update in few mins

Reply:
yes that will be better but I would suggest creating a fresh PR as it will have proper commits, if you ``` git revert ``` it will show in commits.

Reply:
yeah sure

Reply:
Yeah sure will squashing those commits wait